### ISSUE TITLE
Gcc isn't part of the dependencies

### ISSUE THREAD

**Issue by thinkingerrol** [dlang-snaps/dmd.snap#25]
I'm assuming %title% is the problem, because `sudo apt install gcc` solves the
```txt
$ dmd hello.d
cc: No such file or directory
Error: linker exited with status 255
```
on my freshly installed Ubuntu 18.04.1 desktop inside VirtualBox. The only steps done after installing Ubuntu (and before the dmd error message) were these:
```sh
sudo snap install vscode --classic
sudo snap install dmd --classic
sudo snap install ldc2 --classic
```
followed by installing `D language utility extension pack 0.3.0` within vscode.

**Comment by WebDrake** [comment]
Yes, the snap package assumes gcc is available on the host system for linking.

I'm a bit reluctant to bundle it as part of the snap package itself, because that adds multiple complications, although it may be worth including a mention of the gcc requirement in the documentation.
